src/runtime_src/core/tools/common/TestRunner.cpp

```diff
@@ -407,13 +407,27 @@ std::string
 TestRunner::findXclbinPath( const std::shared_ptr<xrt_core::device>& _dev,
                 boost::property_tree::ptree& _ptTest)
 {
+  auto xclbin_name = _ptTest.get<std::string>("xclbin", "");
+  std::string xclbin_path;
+#ifdef _WIN32
+  boost::ignore_unused(_dev);
```

Comment:
It has been in C++ for a decade
```suggestion
  std::ignore = _dev;
```

Reply:
fixed